src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramOptionsTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telegram\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Bridge\Telegram\TelegramOptions;
+
+final class TelegramOptionsTest extends TestCase
+{
+    public function testAnswerCallbackQuery()
+    {
+        $sut = new TelegramOptions();
```

Comment:
```suggestion
        $options = new TelegramOptions();
```
As far as I know, we don't use `SUT` in the Symfony codebase